testing/testing.gni

```diff
@@ -254,10 +266,21 @@ template("test_fixtures") {
   # If a Dart file is specified, snapshot it and place it in the generated
   # assets directory.
   if (defined(invoker.dart_main)) {
+    if (defined(invoker.use_target_as_artifact_prefix) &&
```

Comment:
Please document the new parameter up in the comment with `fixtures` and `dart_main`.

Reply:
Done